components/esp_eth/src/esp_eth_phy_ksz80xx.c

```diff
@@ -427,6 +452,36 @@ esp_eth_phy_t *esp_eth_phy_new_ksz8041(const eth_phy_config_t *config)
     return ret;
 }
 
+esp_eth_phy_t *esp_eth_phy_new_ksz8061(const eth_phy_config_t *config)
```

Comment:
If the KSZ8061 and KSZ8081 are the same we could also consider some kind of simplification of this function.

Reply:
Ok, well. Let me mention here that I tried to adapt my implementation in the way all of the previous impementations in this file were done. And there was done a lot of copy paste of the same code. So I thought this is intended by design.

Note that the functions `esp_eth_phy_new_ksz8041`, `esp_eth_phy_new_ksz8061`, `esp_eth_phy_new_ksz8081` all share the same implementation (instead of the vendor model). This already leaded to the bug #7869.

So to prevent code duplication, I have to refactor all functions, not just mine. For example create a static `esp_eth_phy_new_ksz80xx (int vendor_model)` that is used by these functions. Otherwise, the implementations in the file are inconsistent.

Reply:
Done